src/parquet/file/reader.cc

```diff
@@ -68,7 +68,7 @@ ParquetFileReader::ParquetFileReader() : schema_(nullptr) {}
 ParquetFileReader::~ParquetFileReader() {}
 
 std::unique_ptr<ParquetFileReader> ParquetFileReader::OpenFile(const std::string& path) {
-  std::unique_ptr<LocalFileSource> file(new LocalFileSource());
+  std::unique_ptr<MemoryMapSource> file(new MemoryMapSource());
```

Comment:
We should not hard-code using memory mapping (it is not that advantageous in our use case). You can add a flag to OpenFile() to select between LocalFileSource and MemoryMapSource.